Assets/Scripts/Actors/ActorComponents/PlayerActorStats.cs

```diff
@@ -24,6 +24,10 @@ public class StatObject
 	public Image currentImage = null;
 	public Image maxImage = null;
 	public float statToScaleRatio = 0.0f;
+	public float rechargeDelayTime = 0.0f; // Seconds
+
+	float _rechargeTimer = 0.0f; // Made a property so that it doesn't show up in the inspector where it will be confusing but will show in debug mode.
+	public float rechargeTimer { get { return _rechargeTimer; } set { _rechargeTimer = value; } }
```

Comment:
This would probably be an appropriate use for the fnacy new `[ReadOnly]` attribute.